mods/ra/weapons/ballistics.yaml

```diff
@@ -49,107 +26,69 @@
 		ImpactSounds: splash9.aud
 		ValidImpactTypes: Water
 
-105mm:
-	ReloadDelay: 70
-	Range: 4c768
-	Report: cannon1.aud
-	Burst: 2
-	BurstDelay: 4
+25mm:
+	Inherits: ^Cannon
```

Comment:
Which change do you mean? I disabled the explosion sounds for the 25mm weapon, so it matches bleed.